src/fill_pg_plugin.cpp

```diff
@@ -266,40 +321,65 @@ struct fpg_session : connection_callbacks, std::enable_shared_from_this<fpg_sess
+      t.exec("create table " + converter.schema_name + ".permissions " + R"((permission_number BIGINT PRIMARY KEY, action_number BIGINT, actor CHAR(12), permission CHAR(12)))");
+      t.exec("create table " + converter.schema_name + ".abis " + R"((abi_number BIGINT PRIMARY KEY, action_number BIGINT, account CHAR(12), abi TEXT))");
+
+      // create chintai indices
+      t.exec("create index block_index_timestamp on " + converter.schema_name + ".blocks" + R"((timestamp))");
+      t.exec("create index block_index_id on " + converter.schema_name + ".blocks" + R"((block_id))");
+      t.exec("create index transaction_index_block on " + converter.schema_name + ".transactions" + R"((block_number))");
+      t.exec("create index action_transaction on " + converter.schema_name + ".actions" + R"((transaction_number))");
+      t.exec("create index action_receiver on " + converter.schema_name + ".actions" + R"((receiver))");
+      t.exec("create index action_account on " + converter.schema_name + ".actions" + R"((action_account))");
+      t.exec("create index action_name on " + converter.schema_name + ".actions" + R"((action_name))");
+      t.exec("create index action_data_action_number on " + converter.schema_name + ".action_data" + R"((action_number))");
+      t.exec("create index action_data_key on " + converter.schema_name + ".action_data" + R"((key))");
+      t.exec("create index action_data_value on " + converter.schema_name + ".action_data" + R"((value))");
+      t.exec("create index table_rows_account on " + converter.schema_name + ".table_rows" + R"((account))");
+      t.exec("create index table_rows_scope on " + converter.schema_name + ".table_rows" + R"((scope))");
+      t.exec("create index table_rows_table on " + converter.schema_name + ".table_rows" + R"((table_name))");
+      t.exec("create index table_rows_pk on " + converter.schema_name + ".table_rows" + R"((primary_key))");
+      t.exec("create index table_row_data_table_row_number on " + converter.schema_name + ".table_row_data" + R"((table_row_number))");
+      t.exec("create index table_row_data_block on " + converter.schema_name + ".table_row_data" + R"((block_number))");
+      t.exec("create index table_row_data_key on " + converter.schema_name + ".table_row_data" + R"((key))");
+      t.exec("create index table_row_data_value on " + converter.schema_name + ".table_row_data" + R"((value))");
+      t.exec("create index permissions_action_number on " + converter.schema_name + ".permissions" + R"((action_number))");
+      t.exec("create index permissions_actor on " + converter.schema_name + ".permissions" + R"((actor))");
+      t.exec("create index abis_action_number on " + converter.schema_name + ".abis" + R"((action_number))");
+      t.exec("create index abis_account on " + converter.schema_name + ".abis" + R"((account))");
```

Comment:
maybe lines 354-375 can be shortened through looping a list? seems repetitive

Reply:
@shannendorothee This is definitely a good suggestion - I could for instance create a map, then iterate through the map with a loop function. But I think in this case the map would end up using up just as many lines of code as it is right now, in order to maintain readability. So for now I think I'll leave it as it is

Reply:
Updated.